src-ghc/Pact/Types/Command.hs

```diff
@@ -66,6 +70,11 @@ import Pact.Types.Hash
 import Pact.Parse
 import Pact.Types.RPC
 
+-- | Command is the signed, hashed envelope of a Pact execution instruction or command.
+-- In 'Command ByteString', the 'ByteString' payload is hashed and signed; the ByteString
+-- being the JSON serialization of 'Payload Text', where the 'Text' is the pact code; when
+-- executed this is parsed to 'ParsedCode'.
+-- Thus, 'Command (Payload ParsedCode)' is the fully executable specialization.
 data Command a = Command
```

Comment:
Since 'a' can only be Text or ByteString, what about using a typeclass constraint on functions that use Command to make it clearer?

Reply:
No, it can be `ByteString` or `Payload ParsedCode`

Reply:
Plus it would only be a marker. It might have value to make a type family for it?